tests/storage/export.go

```diff
@@ -1278,13 +1250,13 @@ var _ = SIGDescribe("Export", func() {
 		}
 
 		vm := renderVMWithRegistryImportDataVolume(cd.ContainerDiskCirros, sc)
+		vm = createVM(vm)
 		if libstorage.IsStorageClassBindingModeWaitForFirstConsumer(sc) {
 			// In WFFC need to start the VM in order for the
 			// dv to get populated
-			vm.Spec.RunStrategy = virtpointer.P(v1.RunStrategyAlways)
+			vm = libvmops.StartVirtualMachine(vm)
+			vm = libvmops.StopVirtualMachine(vm)
```

Comment:
There is a hidden logic change here which IMO goes in the wrong direction.
We prefer VMs to be created with a spec that starts them immediately, avoiding more API calls.

But the change here, per what I understand, is first creating the VM in a non-running state, then patches it to start running. The original code is optimized to start as running in the first place.

Reply:
Now `StopVirtualMachine` uses the verb `Stop` which fails when the VM is not started, so being explicit about starting it helps address this issue. Another option would be to add a get in `StopVirtualMachine` and check if vm is started before stopping it, but I think that's more costly than being explicit in some export tests (we also only call this when using wffc). wdyt?

Reply:
I did not know the `Stop` verb fails if it is nor already running, that sounds a bad API (it is raceful).

Is there a specific error that  is received in such a case? We should be able to identify and ignore such errors (because they are really not errors).

Reply:
Yeah, it returns `VM is not running`. I can catch the error string and ignore the error in `StopVirtualMachine`.